src/viam/trajex/totg/trajectory.cpp

```diff
@@ -292,6 +290,10 @@ enum class integration_event : std::uint8_t {
         // Examine the current segment for switching points where the path velocity limit curve, s_dot_max_acc(s),
         // is continuous, but not differentiable, per VII-A Case 2 in the paper (Kunz & Stilman equation 39). These
         // switching points occur on the interior of circular segments for joint extrema where f'_i(s) = 0.
+        //
+        // TODO(RSDK-12979): The paper says that at these points, we must use zero acceleration, but we do not return.
+        // that information in any meaningful way. Currently, a caller who obtains this type of switching point will
+        // erroneously use the local max acceleration, which would lead into the infeasible region.
```

Comment:
Some thoughts:
We have two options:
(1) We just return 0 because we know that will work (but this might mean relaxing constraints in the algorithm itself)
(2) We return the non-zero value and the caller knows it needs to be switched to zero (but this will mean more special casing logic)